File: School source codes/Lab 8/Computeareas.cs
using System;
using System.Collections.Generic;
using System.Linq;

class MainClass {
  public static double SquareArea (double x) {
    double k = x*x;
    return k;
  }
  public static double RectArea (double x,double y) {
    double k = x*y;
    return k;
  }
  public static double CircleArea (double x) {
    double k = (Math.PI)*(x*x);
    return k;
  }
  public static double TriArea (double x, double y) {
    double k = (x*y)/2;
    return k;
  }
  public static void Main (string[] args) {
    int square,rect1,rect2,circle,tri1,tri2;
    Console.Write("enter your square side length: ");
    square = Convert.ToInt32(Console.ReadLine());
    Console.WriteLine("Square's area: " + SquareArea(square));
    Console.WriteLine();
    Console.Write("enter your rectangle side width: ");
    rect1 = Convert.ToInt32(Console.ReadLine());
    Console.Write("enter your rectangle side height: ");
    rect2 = Convert.ToInt32(Console.ReadLine());
    Console.WriteLine("Rectangles area: " + RectArea(rect1,rect2));
    Console.WriteLine();
    Console.Write("enter your circle radius: ");
    circle = Convert.ToInt32(Console.ReadLine());
    Console.WriteLine("Circle's area: " + CircleArea(circle));
    Console.WriteLine();
    Console.Write("enter your triangle base: ");
    tri1 = Convert.ToInt32(Console.ReadLine());
    Console.Write("enter your triangle height: ");
    tri2 = Convert.ToInt32(Console.ReadLine());
    Console.WriteLine("Tirangle's area: " + TriArea(tri1,tri2));
  }
}

File: School source codes/Lab 8/Computeareas.rb
def SquareArea(x)
  k = x * x
end
def RectArea(x,y)
  k = x * y
end
def CircleArea(x)
  k = ((Math::PI) * ( x * x))
end
def TriArea(x, y)
  k = (x * y ) / 2
end
print "enter your square side length: "
square = gets.chomp.to_i
puts "Square's area: #{SquareArea(square)}"
puts "\n"
print "enter your rectangle side width: "
rect1 = gets.chomp.to_i
print "enter your rectangle side height: "
rect2 = gets.chomp.to_i
puts "Rectangles area: #{RectArea(rect1,rect2)}"
puts "\n"
print "enter your circle radius: "
circle = gets.chomp.to_i
puts "Circle's area: #{CircleArea(circle)}"
puts "\n"
print "enter your triangle base: "
tri1 = gets.chomp.to_i
print "enter your triangle height: "
tri2 = gets.chomp.to_i
puts "Tirangle's area: #{TriArea(tri1,tri2)}"